start.py

```diff
@@ -259,7 +276,46 @@ def _remove_files(nodes, files, quiet):
         node.execute(command=command, quiet=quiet)
 
 
+def _install_atlas_streamsets_model(node, custom_stage_lib, quiet):
+    logger.debug('Installing StreamSets Atlas model ...')
+    atlas_custom_stage_lib_name, atlas_custom_stage_lib_version = custom_stage_lib.split(',')
+    atlas_custom_stage_lib_version = ('{}-SNAPSHOT'.format(atlas_custom_stage_lib_version.rsplit('-latest', 1)[0])
```

Comment:
Is this logic right? I think we have some places where we might use `latest` even to refer to a non-SNAPSHOT (e.g. after a release) so perhaps we should be careful.